src/services/api.tsx

```diff
@@ -130,23 +131,20 @@ export const getPatientsPerQuery = async (
    * TODO : for now, this function is limited by the server response to 500 resources. Must find a way to improve it (searching computation done by the server ideally ?)
    */
   let bundles: Bundle[] = [];
-  let params: string;
+  let params: string = "";
 
   if (searchName) {
-    params = "";
-
-    searchName.split(" ").map((x: string) => {
-      params += "&name=" + x;
-      return params;
-    });
+    params += searchName
+      .split(" ")
+      .map((x: string) => `&name=${x}`)
+      .join();
     let bundlePatient = await makeRequest("Patient", false, params, 10000);
 
     let entries = bundlePatient.entry;
-    params = "";
-    searchName.split(" ").map((x: string) => {
-      params += "&identifier=" + x;
-      return params;
-    });
+    params = searchName
```

Comment:
`params =` or `params +=`?

Reply:
We are doing here a second call to get patients per id, so we reinitialize the parameters